File: backend/src/main/java/org/jdk/project/dto/platform/PlatformBootstrapDto.java
package org.jdk.project.dto.platform;

import java.util.List;
import java.util.Map;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PlatformBootstrapDto {
  private String projectCode;
  private String projectName;
  private String customerName;
  private String systemTitle;
  private String brandTitle;
  private String logoUrl;
  private String templateCode;
  private String templateName;
  private String homePath;
  private String themeColor;
  private Map<String, Boolean> modules;
  private Map<String, String> terms;
  private List<PlatformMenuDto> menus;
  private Map<String, PlatformPageDto> pages;
}

File: backend/src/main/kotlin/org/jdk/project/dto/platform/PlatformBootstrapDto.kt
package org.jdk.project.dto.platform

data class PlatformBootstrapDto(
    var projectCode: String? = null,
    var projectName: String? = null,
    var customerName: String? = null,
    var systemTitle: String? = null,
    var brandTitle: String? = null,
    var logoUrl: String? = null,
    var templateCode: String? = null,
    var templateName: String? = null,
    var homePath: String? = null,
    var themeColor: String? = null,
    var modules: Map<String, Boolean>? = null,
    var terms: Map<String, String>? = null,
    var menus: List<PlatformMenuDto>? = null,
    var pages: Map<String, PlatformPageDto>? = null,
)
